ckanext/og_datatablesview/utils.py

```diff
@@ -0,0 +1,181 @@
+# encoding: utf-8
+
+from six.moves.urllib.parse import urlencode
+from six import text_type
+
+from ckan.common import json
+from ckan.plugins.toolkit import get_action, request, h, requires_ckan_version
+from ckan.exceptions import CkanVersionException
+
+
+def remove_null_values(records):
```

Comment:
name 'remove_null_values' is not connected to anything that happens in function. 'replace_none_with_empty_string' represents function better.

Reply:
renamed the function